BUILD.txt

```diff
@@ -1,5 +1,7 @@
 Doxygen uses cmake (http://www.cmake.org/) to build executables for various platforms.
 At least cmake version 3.14 is required.
+G++, Python, Flex, and Bison are also required.
```

Comment:
I think it would be better to refer to the manual (chapter Installation) otherwise the same information is scattered on multiple places.
The reference to docker would be OK.

Reply:
OK, updated